### PR TITLE
fix: minor changes

### PR DESCRIPTION
**Pull request by hpsing**
* minor fix with bigInt check

### CODE REVIEW

File: bigint.go [hunk 1]
```diff
@@ -9,10 +9,14 @@ import (
...
 func ToBigInt(b *protov1.BigInt) (*big.Int, error) {
+	if b == nil || len(b.GetValue()) == 0 {
+		return nil, errors.New("input cannot be nil or empty")
+	}
+
 	result := new(big.Int)
 	result = result.SetBytes(b.GetValue())
 
-	if len(result.Bits()) == 0 {
+	if result.Sign() == 0 {
```
Comment (adelowo) [inline bigint.go]: This does not handle negative numbers like previously. 

```suggestion
        if result.Sign() != 1 {
```

Comment (hpsing) [reply bigint.go]: ohh nice catch thanks @adelowo , even `!=1` also won't this condition only allows positive numbers and incorrectly handles any non-positive values, including zero, as errors.

Comment (blewater) [reply bigint.go]: `if result.Sign() <= 0` right?

File: proto/v1/asset.proto [hunk 2]
```diff
@@ -23,6 +23,7 @@ enum ProcessingStatus {
 // BigInt represents a large number
 message BigInt {
   bytes value = 1;
+  bool negative = 2;
```
Comment (blewater) [inline proto/v1/asset.proto]: Negative values are rejected -> https://github.com/blndgs/model/blob/1971b6e3247a4427e68283f288d53d7a773f35bc/bigint.go#L37 and BigInt only carries positive values, this is redundant, and the artificial BigInt negative tests can be removed.

**Review by blewater — APPROVED** [review]
LGTM!

**Pull request merged**